addons/event/tests/test_event_mail_schedule.py

```diff
@@ -89,12 +89,14 @@ def test_event_mail_schedule(self):
 
         # create some registrations
         with freeze_time(now), self.mock_mail_gateway():
-            reg1 = self.env['event.registration'].with_user(self.user_eventuser).create({
+            reg1 = self.env['event.registration'].create({
+                'create_date': now,
```

Comment:
Don't think you have to specify the create_date, the ORM should have its time frozen and compute it correctly.
Same applies for other create calls.

Reply:
Well you end up with create date linked to SQL date. I had to do the same trick elsewhere ...